.github/actions/build/action.yaml

```diff
@@ -10,30 +10,30 @@ runs:
       - name: Get depot tools
         run: git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git
         shell: bash
-      - name: Add directory to PATH
+      - name: Add depot tools to PATH
         run: echo "$GITHUB_WORKSPACE/depot_tools" >> $GITHUB_PATH
         shell: bash
       - name: Generate gclient file
-        run: gclient config --name=src https://github.com/${{ github.repository }}
+        run: gclient config --name=src https://github.com/${{ github.repository }} --custom-var=download_remoteexec_cfg=True --custom-var='rbe_instance="projects/cobalt-actions-prod/instances/default_instance"'
```

Comment:
nit: Options are prescribed _before_ the URL:
`Usage: gclient.py config [options] [url]`